src/main/java/org/rulez/demokracia/pdengine/assurance/ADAAssuranceManager.java

```diff
@@ -0,0 +1,15 @@
+package org.rulez.demokracia.pdengine.assurance;
```

Comment:
I think something went wrong on rebase. This file is not related to the modification and has been existed earlier.